SOLIDHomework.Core/Model/ShoppingCart.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SOLIDHomework.Core.Model
+{
+    public class ShoppingCart
+    {
+        public string Country { get; set; }
+        public List<OrderItem> Items { get; set; } = new List<OrderItem>();
```

Comment:
fields should be private, if you need to get a values of them, implement properties or methods

Reply:
I changed the variables to private